build/storage/tests/it/ptf_tests/tests/test_hot_plug.py

```diff
@@ -0,0 +1,166 @@
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+from ptf.base_tests import BaseTest
+from system_tools.config import TestsConfig
+from system_tools.errors import CommandException
+from system_tools.test_platform import KVMPlatformsFactory
+
+
+class TestMinHotPlug(BaseTest):
+    def setUp(self):
+        self.platforms_factory = KVMPlatformsFactory()
+        self.storage_target_platform = (
+            self.platforms_factory.create_storage_target_platform()
+        )
+        self.ipu_storage_platform = self.platforms_factory.create_ipu_storage_platform()
+        self.host_target_platform = self.platforms_factory.create_host_target_platform()
+        self.tests_config = TestsConfig()
+
+    def runTest(self):
+        self.assertTrue(
+            self.storage_target_platform.is_port_free(self.tests_config.nvme_port)
+        )
+        self.storage_target_platform.create_subsystem(self.tests_config)
+        self.assertFalse(
+            self.storage_target_platform.is_port_free(self.tests_config.nvme_port)
+        )
+        self.assertTrue(
+            self.storage_target_platform.is_app_listening_on_port(
+                "spdk_tgt", self.tests_config.nvme_port
+            )
+        )
+
+        volumes = self.storage_target_platform.create_ramdrives(
+            self.tests_config.min_ramdrive,
+            self.tests_config,
+        )
+        self.assertEqual(len(volumes), self.tests_config.min_ramdrive)
+
+        self.assertEqual(
+            self.host_target_platform.get_number_of_virtio_blk_devices(), 0
+        )
+        devices_handles = (
+            self.ipu_storage_platform.create_virtio_blk_devices_sequentially(
+                self.host_target_platform.get_service_ip_address(),
```

Comment:
this port is no needed in create virtio blk

Reply:
It can be a good example how we can easily mess up with parameters if we just hard code them here and there.
It is used, but hard-coded [here ](https://github.com/intelfisz/ipdk/blob/db17982f817c3321d9d6d084b4b0eaf9070b794e/build/storage/tests/it/ptf_tests/system_tools/docker.py#L158) and [here](https://github.com/intelfisz/ipdk/blob/db17982f817c3321d9d6d084b4b0eaf9070b794e/build/storage/tests/it/ptf_tests/system_tools/docker.py#L177) as `50051`

Reply:
changed